src/test/java/org/jpeek/web/ReportsTest.java

```diff
@@ -46,20 +46,21 @@
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
 public final class ReportsTest {
 
-    @Before
+    @BeforeEach
     public void weAreOnline() {
         try {
             new TextOf(new URL("http://www.jpeek.org/")).asString();
         } catch (final IOException ex) {
             Logger.debug(this, "We are not online: %s", ex.getMessage());
-            Assume.assumeTrue(false);
+            Assumptions.assumeTrue(false);
         }
     }
 
     @Test
     public void rendersOneReport() throws Exception {
-        final BiFunc<String, String, Func<String, Response>> reports =
-            new Reports(Files.createTempDirectory("x"));
+        final BiFunc<String, String, Func<String, Response>> reports = new Reports(
+            Files.createTempDirectory("x")
```

Comment:
@HDouss Why not using `org.junit.jupiter.api.io.TempDir` ?

Reply:
@fanifieiev Fixed it! I did not do it as it is not JUnit 4 api, so I did not take that as migration. We have too many of these, and I submitted a separate issue for this: #394

Reply:
@HDouss Ok